modules/administration/pages/virtualization.adoc

```diff
@@ -315,6 +315,10 @@ Apart from being able to serve as a host for {kvm} or {xen} guests, which are ma
 ** Execute the bootstrap script to register the {vmhost}.
 +
 
+[IMPORTANT]
+====
+After bootstrap, Salt minions need to be restarted in order to get instant virtual machines changes reflected in {productname}.
```

Comment:
```suggestion
After bootstrap, Salt minions need to be restarted in order to get instant virtual machine changes reflected in {productname}.
```
Sounds better, IMO (but could be wrong).

Reply:
I'll have a go, too ;)

```suggestion
Salt minions must be restarted after bootstrapping, in order for instant virtual machine changes to be reflected in {productname}.
```

Reply:
s/bootstrapping/enabling Virtualization Host entitlement/ would be more accurate